mcode/request-generator
# Rework etasu endpoint to call new standards based endpoint

## Describe your changes

Replace existing etasu calls to make a new standards based etasu call. 

To test:
Select a few medications and go through the workflow to have no etasu, pending etasu, and approved etasu. Button should be reflected, and when clicked the modal should show which rems requirements have and have not been met. 

Also make sure to test patient portal, should show all medications for the patient. 

## Issue ticket number and Jira link

https://jira.mitre.org/browse/REMS-646 

## Checklist before requesting a review

## Checklist for conducting a review

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.
